irohad/torii/impl/command_service_impl.cpp

```diff
@@ -35,19 +35,20 @@ namespace iroha {
           tx_presence_cache_(std::move(tx_presence_cache)),
           log_(std::move(log)) {
       // Notifier for all clients
-      status_subscription_ =
-          status_bus_->statuses().subscribe([this](auto response) {
+      status_subscription_ = status_bus_->statuses().subscribe(
+          // TODO mboldyrev IR-426 capture this, wait termination in destructor
```

Comment:
>capture this, wait termination in destructor

Not sure if blocking is the rx way to solve this. I would rather rephrase it as "research and list the approaches to this problem".